internal/x/format/base_visitor.go

```diff
@@ -162,7 +164,9 @@ func (v *baseVisitor) pInnerLiteral(name string, literal proto.Literal, suffix s
 	}
 	// TODO: this is a good example of the reasoning for https://github.com/uber/prototool/issues/1
 	if len(literal.Array) == 0 && len(literal.OrderedMap) == 0 {
-		v.P(prefix, literal.SourceRepresentation(), suffix)
+		if source := literal.SourceRepresentation(); source != "" {
```

Comment:
It's pretty unclear what this branch is checking / doing. Could we add some more context here, perhaps with a new helper function or in-line description?

Reply:
Can I just add a comment that says “SourceReprrsentation() returns an empty string if the literal is empty, and if empty, we do not want to print the key or empty value”?